src/map/handler/Map.Keyboard.js

```diff
@@ -47,6 +47,13 @@ export const Keyboard = Handler.extend({
 			container.tabIndex = '0';
 		}
 
+		// add aria-attribute for keyboard shortcuts to the container
+		let keyShortcuts = [];
+		for (const key in this.keyCodes) {
+			keyShortcuts = keyShortcuts.concat(this.keyCodes[key]);
+		}
+		container.ariaKeyShortcuts = keyShortcuts.join(' ');
```

Comment:
We can simplify this:
```suggestion
		container.ariaKeyShortcuts = Object.values(this.keyCodes).flat().join(' ') ;
```

Reply:
If we would like to make sure that no shortcut exsits multiple times, we could wrap it inside a `Set` but I think we have a differnet problem, if a keybinding is used for different commands.

`[...new Set(Object.values(keyCodes).flat())].join(' ') `

Reply:
I've always preferred the for...in-syntax, but I'll convert it to the one-liner. I agree the set may be overkill for something that probably won't happen, and in my opinion the one-liner starts will get too terse if it does five things at once.

Reply:
I've implemented the change and squashed the commits